### ISSUE TITLE
NPE if user is trying to register new user in NGB that was not configured for RBAC.

### ISSUE THREAD

**Issue by vpolyakov88** [epam/NGB#251]
1. Run NGB from jar file without any environment settings for RBAC.
2. Run CLI and configure server.
3. Do CLI command `ngb ru "{user_name@mail.com}"

exp.: handled error is displayed.
act.: unhandled error is displayed.
```
Exception in thread "main" java.lang.NullPointerException
        at com.epam.ngb.cli.manager.command.handler.http.CreateUserHandler.parseAndVerifyArguments(CreateUserHandler.java:99)
        at com.epam.ngb.cli.manager.command.CommandManager.run(CommandManager.java:90)
        at com.epam.ngb.cli.app.Application.createAndLaunchCommandManager(Application.java:258)
        at com.epam.ngb.cli.app.Application.run(Application.java:220)
        at com.epam.ngb.cli.app.Application.main(Application.java:206)
```

**Comment by SilinPavel** [comment]
Fixed in 31afca2a. @vpolyakov88 test it please. Note that now all role-model related cli command will throw if the server not supports role model.

**Comment by vpolyakov88** [comment]
tested
